third_party/gpus/cuda_configure.bzl

```diff
@@ -1295,6 +1312,19 @@ def _create_local_cuda_repository(repository_ctx):
     else:
         copy_rules.append("filegroup(name = 'cudnn-include')\n")
 
+    # Copy cublas headers if not installed to CUDA_TOOLKIT_PATH.
+    if not any([file.endswith("cublas_v2.h") for file in included_files]):
+        cublas_header_dir = "/usr/include"
```

Comment:
Would it be possible to check where these files can be found in configure.py, similar to how we do it for cuDNN and TensorRT? I'm suspecting the path will be different on Windows.